example/take_screenshot.js

```diff
@@ -0,0 +1,17 @@
+const bin = require('ffmpeg-binaries').ffmpegPath();
+const cp = require('child_process');
+const ytdl = require('..');
+
+const takeScreenshot = (url, outFile, position) => new Promise((resolve, reject) => {
+  ytdl.getInfo(url).then(info => {
+    let format = info.formats[0];
```

Comment:
Would prefer [`ytdl.chooseFormat()`](https://github.com/fent/node-ytdl-core#ytdlchooseformatformats-options) here to give users a more flexible option, maybe with `quality: 'highestvideo'`